lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -243,6 +244,27 @@ def extract_student(student, features):
     return [extract_student(student, features) for student in may_enroll_and_unenrolled]
 
 
+def get_proctored_exam_results(course_key, features):
+    """
+    Return info about proctored exam results in a course as a dict.
+    """
+    exam_attempts = get_all_exam_attempts(course_key)
```

Comment:
Let's move this line down below the internal function definition, closer to where it is invoked